Doc/library/enum.rst

```diff
@@ -514,6 +525,65 @@ Data Types
    Using :class:`auto` with :class:`IntFlag` results in integers that are powers
    of two, starting with ``1``.
 
+.. class:: EnumCheck
+
+   *EnumCheck* contains the options used by the :func:`verify` decorator to ensure
+   various constraints; failed constraints result in a :exc:`TypeError`.
+
+   .. attribute:: UNIQUE
```

Comment:
Is there any guidance on when to use `@unique` vs `@verify(UNIQUE)`?

Reply:
The only difference between the two is `unique` raises `ValueError` and `verify` raises `TypeError`.  I don't see `unique` going away, so it's pretty much personal preference.

Reply:
Changed that so both raise `ValueError`.  I added `UNIQUE` to `verify()` for two reasons:
- so multiple checks would not require multiple decorators
- so `unique()` could eventually go away (not that it will any time soon)